src/dist/bernoulli.rs

```diff
@@ -40,6 +40,18 @@ pub struct Bernoulli {
     p: f64,
 }
 
+impl Parameterized for Bernoulli {
+    type Parameters = f64;
+
+    fn emit_params(&self) -> Self::Parameters {
+        self.p()
+    }
+
+    fn from_params(params: Self::Parameters) -> Self {
+        Self::new_unchecked(params)
+    }
```

Comment:
Since the implementations are proper `struct`s, would it be better to drop this and `emit_params` and just use the standard `From` trait?